maths/signum.py

```diff
@@ -7,12 +7,21 @@ def signum(num: float) -> int:
     """
     Applies signum function on the number
 
-    >>> signum(-10)
+    Custom test cases:
+    >>> signum(-20)
     -1
-    >>> signum(10)
+    >>> signum(20)
     1
     >>> signum(0)
     0
+    >>> signum("a")
+    0
```

Comment:
```suggestion
    Traceback (most recent call last):
        ...
    TypeError: '<' not supported between instances of 'str' and 'int'
```

Reply:
Check I've added changes

Reply:
Please revert this change.  The code CORRECTLY raises a TypeError if you try to give it a string instead of a number.  This is MUCH better than sweeping that error under the rug and _pretending_ the answer is zero when it is NOT.